src/rollup/po2i18n.js

```diff
@@ -0,0 +1,24 @@
+const createFilter = require('@rollup/pluginutils').createFilter;
+const parsePoFile = require('../parsePoFile/parsePoFile');
+
+module.exports = function po2I18next ({
+    include,
+    exclude
+}) {
+    const filter = createFilter(include, exclude);
+
+    return {
+        transform(originalCode, id) {
+            if (id.slice(-3) !== '.po' || !filter(id)) {
+                return;
+            }
+
+            const { translations, p11nRules } = parsePoFile(originalCode);
+
+            return {
+                code: `export default ${translations}; export const p11nRules = ${p11nRules}`,
```

Comment:
Maybe we could check if `translations` and `p11nRules` aren't empty strings and default to `{}` and `()=>{}` respectively to ensure the file can be parsed. (ie, we don't want to generate `export default ; export const p11nRules = `)

Reply:
Translations would aways be an object, because `pofile` returns empty array if there's not entries, and I've added fallback to default implementation in 9db71a7074f9077dbe6fb76f95af20c50ec7d9a4